anasymod/structures/module_clk_manager.py

```diff
@@ -72,17 +73,17 @@ def __init__(self, scfg: StructureConfig):
 	        #((0.25/(`EMU_CLK_FREQ))*1s);
 	        emu_clk_2x_state = ~emu_clk_2x_state;
 	    end
-	    
+	end
+
 {% for clk in subst.independent_clks %}
+    initial begin
+        {{clk.name}}_state = 1'b0;
 	    forever begin
-	        // TODO: investigate whether these clocks periods should be
-	        // related to `EMU_CLK_FREQ / `DT_MSDSL or not (since they are
-	        // independent)
-	        #(({{0.50 * (subst.emu_clk.freq / clk.freq)}}*`DT_MSDSL)*1s);
+	        #(({{0.5/clk.freq}})*1s);

```

Comment:
With this update, independent clocks now toggle at their specified frequency in simulation (since they operate outside of the time manager).